2333/2/libs/rapper.rb

```diff
@@ -0,0 +1,30 @@
+require 'russian_obscenity'
+
+# class Rapper
+class Rapper
+  attr_reader :name, :battles, :number_of_bad_words
+  ROUNDS_ON_BATTLE = 3
+
+  def initialize(name, battles)
+    @name = name
+    @battles = battles
+    @number_of_bad_words = bad_words
```

Comment:
Why do we need this instance variable? What if we just rename `#bad_words` method to `#number_of_bad_words`?